applications: Matter Bridge: added support for Generic Switch (#13318)
nrfconnect/sdk-nrf: applications/matter_bridge/README.rst

               :class: highlight

               I: Added device to dynamic endpoint 3 (index=0)
               I: Added device to dynamic endpoint 4 (index=1)
               I: Created 0x100 device type on the endpoint 3
               I: Created 0xf device type on the endpoint 4

            For the LED Button Service and the Environmental Sensor, two endpoints are created:

>             * For the LED Button Service, one implements the OnOff Light Device and the other implements the Generic Switch Device.

Reviewer: This is written OnOff here, but On/Off on Line 210. Which is correct?

               ./chip-tool onoff read on-off *<bridge_node_ID>* *<bridged_device_endpoint_ID>*

            Read the value of the *current-position* attribute from the *switch* cluster using the following command:

            .. parsed-literal::
               :class: highlight

               ./chip-tool switch read current-position *<bridge_node_ID>* *<bridged_device_endpoint_ID>*

>             Note that the Generic Switch implements the momentary switch which, in contrast to the latching switch, remains switched on only as long as the physical button is compressed.

Reviewer: ```suggestion
            Note that the Generic Switch implements the momentary switch.
            This means that, in contrast to the latching switch, it remains switched on only as long as the physical button is pressed.
```
Is it correct to say "implements the momentary switch", or should it be "is implemented as a momentary switch"?

Author: I'm not sure, for me both versions seem correct. I can use the latter, no problem.
